src/ngx_http_upsync_module.c

```diff
@@ -982,8 +982,8 @@ ngx_http_upsync_del_peers(ngx_cycle_t *cycle,
         for (i = 0; i < servers->nelts; i++) {
 
             server = (ngx_http_upstream_server_t *)servers->elts + i;
-            if (ngx_memn2cmp((u_char *) peer->sockaddr, 
-                             (u_char *) server->addrs->sockaddr, len, len) == 0) 
+            if (ngx_memn2cmp((u_char *) peer->sockaddr, (u_char *) server->addrs->sockaddr,
+                             ngx_strlen(peer->sockaddr), ngx_strlen(server->addrs->sockaddr)) == 0) 
```

Comment:
Each line should be not more than 80 characters or too much. One new line?

Other is the same.

Thanks.